tests/realtime/utils.js

```diff
@@ -206,6 +206,22 @@ module.exports = {
     return JSON.parse(JSON.stringify(standardBotData));
   },
 
+  forceDeleteAspectsAndSubjects(done) {
+    tu.db.Event.destroy({
```

Comment:
Why Event here?

Reply:
following the convention. The line below at 226: forceDelete says to tu.db.Event...